packages/ember-testing/tests/adapters_test.js

```diff
@@ -50,3 +50,22 @@ QUnit.test('QUnitAdapter is used by default', function() {
 
   ok(Test.adapter instanceof QUnitAdapter);
 });
+
+QUnit.test('Adapter is used by default (if QUnit is not available)', function() {
+  expect(2);
+
+  var originalQUnit = window.QUnit;
+  delete window.QUnit;
+
+  Test.adapter = null;
+
+  run(function() {
+    App = EmberApplication.create();
+    App.setupForTesting();
+  });
+
+  window.QUnit = originalQUnit;
```

Comment:
Please reset this in the modules `teardown` above.  We need to ensure that this test failing does not completely hose all the rest of the tests by not cleaning up after itself properly.